git_push/push_archive_to_repo.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+# Executable that unpacks an archive, and pushes the content as a git repo
+# Assumes that `git` is on the machine and has push credentials already.
+# Inspired from how Angular publishes snapshots to "*-build" repos:
```

Comment:
nit: also assumes that `tar` is on the machine... it should be on all major linux distros and all on modern Windows releases

Reply:
mention that `MYORG_GHBOT_PAT` needs to be renamed and set as well?

Reply:
nit: also assumes `mktemp` is on the machine